src/webportal/src/app/vc/vc-modal-component.ejs

```diff
@@ -0,0 +1,61 @@
+<!--  Modal component for vc add -->
+  <div class="modal fade vc-model" id="virtualClustersList" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
Set the actual label id to model title or totally remove `aria-labelledby`, it will confuse the screen reader.